project/dependencies.scala

```diff
@@ -27,6 +27,7 @@ object Dependencies {
   object PlayJsonVersion {
     val V27 = PlayJsonVersion("27", "com.typesafe.play", "2.7.4")
     val V28 = PlayJsonVersion("28", "com.typesafe.play", "2.8.2")
+    val V29 = PlayJsonVersion("29", "com.typesafe.play", "2.10.4", supportsScala3 = true)
```

Comment:
The `supportsScala3` flag isn't actually used here - but it doesn't mean it _requires_ Scala 3! Just that, in principle, we could create Scala 3 artifacts for this version.